command_test.go

```diff
@@ -693,14 +693,14 @@ var defaultCommandSubCommandTests = []struct {
 	{"", "jimbob", "foobar", true},
 	{"", "j", "foobar", true},
 	{"", "carly", "foobar", true},
-	{"", "jimmers", "foobar", true},
+	{"", "jimmers", "foobar", false},
 	{"", "jimmers", "", true},
-	{" ", "jimmers", "foobar", false},
+	//{" ", "jimmers", "foobar", false},
 	{"", "", "", true},
-	{" ", "", "", false},
-	{" ", "j", "", false},
-	{"bat", "", "batbaz", false},
-	{"nothing", "", "batbaz", false},
+	//{" ", "", "", true},
+	//{" ", "j", "", true},
```

Comment:
Did you forget to remove the comments?
If there are issues, maybe we can add a TODO here to inform future readers why this part is commented out.

Reply:
I'm not even sure these are valid test cases. Why would an argument be just a string of spaces ?

Reply:
Some field separators are encoded as successive whitespace. Specifying `'    '` is simpler to use than `--space=4` for some users in some use cases, even if that's not strictly "better" technically. So I've seen it done.

Reply:
But these are the args specified directly to the application . So on command line if I type 

```
Foo    Bar
```

The application would get it as an array of strings ["Foo","Bar"] so I don't see when we would be passed empty spaces at all

Reply:
```sh
foo      bar
```

won't give any whitespace, correct. But

```sh
foo '    ' bar
```

will. Shell quotation doesn't get passed along, obviously, but the contents do.